tied-buffer/sycl-2.2/buffer-tags.md

```diff
@@ -54,14 +63,15 @@ class buffer {
 With this interface, a user can specify a specialisation of the buffer class by simply providing one or more values of the following tag enum class to the constructor of a buffer.
 
 ```cpp
-enum class tag {
-  context_bound,
-  mapping,
-  svm
+namespace property {
+  struct map_ptr;
+  struct cl_interop;
+  struct gl_interop;
+  struct svm;  // SYCL 2.2 Only
 };
 ```
 
-Some of these tags will require an additional interface for the buffer class, for example, the `tag::context_bound` tag will need a `get_context()` member function.
+Some of these tags will require an additional interface for the buffer class, for example, the `property::context_bound` tag will need a `get_context()` member function.
```

Comment:
That means the buffer class must construct a new instance of the property with the required values (since we cannot mandate storing the original instance the user created), can you prototype that?